static/docs/command-reference/checkout.md

```diff
@@ -6,7 +6,7 @@ DVC-files.
 ## Synopsis
 
 ```usage
-usage: dvc checkout [-h] [-q | -v] [-d] [-f] [-R]
+usage: dvc checkout [-h] [-q | -v] [-d] [-R] [-f] [--relink]
```

Comment:
It's

```suggestion
dvc checkout [-h] [-q | -v] [-f] [--relink] [-d] [-R]
```

when you run `dvc checkout -h`.

Reply:
I've already changed it in dvc, so it's:
```
usage: dvc checkout [-h] [-q | -v] [-d] [-R] [-f] [--relink]
                    [targets [targets ...]]
```
there noe.

Reply:
Ah OK, cool. Thanks.